site/index.html

```diff
@@ -0,0 +1,684 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <title>Clarity Design System - Where to Find...</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="Directory of example Clarity Design demos.">
+  <meta name="keywords" content="HTML, CSS, JavaScript, Clarity Design">
+  <link rel="dns-prefetch" href="//cdn.skypack.dev">
+  <link rel="icon" href="favicon.ico" type="image/x-icon">
+  <link rel="stylesheet" href="https://unpkg.com/@cds/core@6.0.0-beta.3/global.min.css">
+  <link rel="stylesheet" href="https://unpkg.com/@cds/core@6.0.0-beta.3/list/list.min.css" media="print" onload="this.media='all'">
+  <link rel="stylesheet" href="https://unpkg.com/@cds/core@6.0.0-beta.3/styles/theme.dark.min.css" media="print" onload="this.media='all'">
+  <link rel="stylesheet" href="https://unpkg.com/@cds/city@1.1.0/css/bundles/default.min.css" media="print" onload="this.media='all'">
+  <noscript><style>:not(:defined) { visibility: visible }</style></noscript>
+  <style>
+    :not(:defined) {
+      visibility: hidden
+    }
+
+    pre {
+      margin: 0;
+      background: var(--cds-global-color-construction-50);
+      padding: var(--cds-global-space-6);
+      width: 100%;
+      overflow-x: auto;
+    }
+
+    [spin] { animation: spin 3s infinite }
+    @keyframes spin { from { transform:rotate(0deg) } to { transform:rotate(360deg) } }
+
+    [cds-theme='dark'] pre {
+      background: var(--cds-global-color-construction-900);
+    }
+
+    [cds-theme='dark'] cds-alert-group a {
+      color: var(--cds-global-color-gray-0) !important; /* temp fix */
+    }
+
+    body {
+      padding: 0;
+    }
+
+    main {
+      padding: 36px 24px 36px 64px;
+    }
+
+    main[expanded] {
+      padding: 36px 24px 24px 264px;
+      z-index: -1;
+    }
+
+    cds-navigation {
+      box-shadow: var(--cds-alias-object-shadow-300);
+      overflow: auto;
+      position: fixed;
+      inset: 0 auto 0 0;
+      z-index: 999;
+    }
+
+    cds-navigation:not([expanded]) cds-navigation-start svg {
+      display: none;
+    }
+
+    main {
+      max-width: 1490px;
+    }
+
+    .demos .icon-font {
+      font-size: 18px;
+    }
+
+    .demos li cds-icon {
+      --width: 18px;
+      --height: 18px;
+    }
+  </style>
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-WM2WWGG0K5"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+    gtag('config', 'G-WM2WWGG0K5');
```

Comment:
is this your GA or the team's?

Reply:
Good catch, will remove. The clarity.design domain can be added later